undebt/pattern/interface.py

```diff
@@ -74,10 +76,35 @@ def patterns_from_files(pattern_files):
 
 def load_module(path):
     """Loads a module from its path."""
+    if module_like(path):
+        return _load_module(path)
+
     pattern_name = os.path.splitext(os.path.basename(path))[0]
     return imp.load_source(pattern_name, path)
 
 
+def _load_module(full_name):
+    try:
+        return importlib.import_module(full_name)
+    except ImportError as e:
+        raise e
```

Comment:
Wait, then is there not a way to re-raise an exception? That's what I was going for here